modules/nw-route-specific-annotations.adoc

```diff
@@ -70,6 +70,27 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 Environment variables cannot be edited.
 ====
 
+.Router timeout variables
+
+TimeUnits are represented by a number followed by the unit: us *(microseconds), ms (milliseconds, default), s (seconds), m (minutes), h *(hours), d (days).
+
+The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
+[cols="3*"]
+|===
+| `ROUTER_BACKEND_CHECK_INTERVAL` | 5000ms | Length of time between subsequent liveness checks on backends.
+| `ROUTER_CLIENT_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout period for the client connecting to the route. If the FIN sent to close the connection is not answered within the given time, HAProxy will close the connection. This is harmless if set to a low value and uses fewer resources on the router.
+| `ROUTER_DEFAULT_CLIENT_TIMEOUT` | 30s | Length of time that a client has to acknowledge or send data.
+| `ROUTER_DEFAULT_CONNECT_TIMEOUT` | 5s | The maximum connect time.
+| `ROUTER_DEFAULT_SERVER_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout from the router to the pod backing the route.
+| `ROUTER_DEFAULT_SERVER_TIMEOUT` | 30s | Length of time that a server has to acknowledge or send data.
+| `ROUTER_DEFAULT_TUNNEL_TIMEOUT` | 1h | Length of time for TCP or WebSocket connections to remain open. If you have websockets/tcp connections and any time HAProxy is reloaded, the old HAProxy processes will stay for that period.
```

Comment:
Please double-check this slash.

> In most cases, do not use slashes in text because they can be ambiguous.

> In a few cases, use a forward slash (/) in terms where the slash construction is an industry standard.